tests/LocalDriverTest.php

```diff
@@ -9,24 +9,43 @@
 class LocalDriverTest extends TestCase
 {
     protected static LocalDriver $localDriverClass;
-    protected static string $productionFilePath;
+    protected static string $baseFileName;
+    protected static string $baseFile;
     protected static string $tmpDocumentationFilePath;
     protected static array $tmpData;
 
     public function setUp(): void
     {
         parent::setUp();
 
-        self::$productionFilePath ??= __DIR__ . '/../storage/documentation.json';
-        self::$tmpDocumentationFilePath ??= __DIR__ . '/../storage/temp_documentation.json';
+        $documentationDirectory = config('auto-doc.drivers.local.directory');
+        if (!str_ends_with($documentationDirectory, DIRECTORY_SEPARATOR)) {
```

Comment:
as we discussed in other file - we always know the package configuration, so no need to check it in tests